libs/openchallenges/home/src/lib/statistics-viewer/statistics-viewer.component.ts

```diff
@@ -75,10 +75,29 @@ export class StatisticsViewerComponent implements OnInit, OnDestroy {
                 },
                 // disable default clicking
                 silent: true,
-                // make bar plot rise from left to right instead of rising all together in the same time
+                // make bar plot rise from left to right
+                // instead of rising all together in the same time
                 animationDelay: (dataIndex: number) => dataIndex * 100,
               },
             ],
+            graphic: res.undatedChallengeCount
+              ? {
+                  elements: [
+                    {
+                      type: 'text',
+                      style: {
+                        text:
+                          `*An additional ${res.undatedChallengeCount} challenges ` +
```

Comment:
It might just be me, but this reads awkward - I think because of the double negative?  Maybe reword it to:

"Due to missing start dates, an additional xx challenges are not displayed."

Reply:
I had the same thought on Slack. Here are suggestions:

- *The OC database includes 50 more challenges without known start dates.
- *The OC database includes an extra 50 challenges without known start dates.

Reply:
Thanks both! I will replace text with`*The OC database includes an extra 50 challenges without known start dates.` for now~